src/ResourceGraph/ResourceGraph/help/Search-AzGraph.md

```diff
@@ -169,6 +169,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ProgressAction
```

Comment:
@Pan-Qi  'ProgressAction' is not expected to be generated in help. Would you sync with main and rebuild the module? After that, 'ProgressAction' will be removed.